docs/src/examples/components/Menu/Slots/MenuExampleIconOnly.shorthand.steps.ts

```diff
@@ -0,0 +1,3 @@
+import getSteps from '../commonSteps'
+
+export default getSteps()
```

Comment:
```suggestion
export default getScreenerSteps()
```

It's clear when they defined, but it's clear when we just export them :)

Reply:
done, thanks